themeconfigurator.php

```diff
@@ -498,6 +500,13 @@ public function getContent()
 			Configuration::updateValue('PS_TC_ACTIVE', (int)Tools::getValue('live_conf'));
 			Configuration::updateValue('PS_GRID_PRODUCT', (int)Tools::getValue('grid_list'));
 			Configuration::updateValue('PS_SET_DISPLAY_SUBCATEGORIES', (int)Tools::getValue('sub_cat'));
+			Configuration::updateValue('PS_TC_CUSTOM_CSS', Tools::getValue('PS_TC_CUSTOM_CSS'), true);
+			Configuration::updateValue('PS_TC_CUSTOM_JS', Tools::getValue('PS_TC_CUSTOM_JS'), true);
+
+			// Custom CSS and JS
+			file_put_contents(_PS_THEME_DIR_.'css/autoload/tc_custom.css', Configuration::get('PS_TC_CUSTOM_CSS'));
```

Comment:
What if `Configuration::get('PS_TC_CUSTOM_CSS')` is empty ? We should not create an empty file, should we ? 🤔